root/programs/VB/Samples/WebApp_sample/ProjectX_sample/App_Code/sample/Dao/LayerD.vb

```diff
@@ -118,15 +118,15 @@ Public Class LayerD
     ''' <summary>件数情報を返すSELECTクエリを実行する</summary>
     ''' <param name="testParameter">引数クラス</param>
     ''' <param name="testReturn">戻り値クラス</param>
-    Public Sub SelectCount(ByVal testParameter As TestParameterValue, ByVal testReturn As TestReturnValue)
+    Public Sub SelectCount(testParameter As TestParameterValue, testReturn As TestReturnValue)
         ' ↓DBアクセス-----------------------------------------------------
 
         Dim filename As String = ""
 
-        If (testParameter.ActionType.Split("%"c))(2) = "static" Then
+        If (testParameter.ActionType.Split("%"C))(2) = "static" Then
```

Comment:
@daisukenishino 

When implementing in this way, Visual Studio converts the program as follows forcibly.
```vb
If (testParameter.ActionType.Split("%"c))(2) = "static" Then
```

Therefore, I think that this revision is unnecessary.